deeprankcore/models/operations/parsers/__init__.py

```diff
@@ -0,0 +1,112 @@
+import os
+import logging
+from deeprankcore.models.structure.atom import Atom
+from deeprankcore.models.structure.residue import Residue
+from deeprankcore.models.operations.parsers.top import TopParser
+from deeprankcore.models.operations.parsers.patch import PatchParser, PatchActionType
+from deeprankcore.models.operations.parsers.residue import ResidueClassParser
+from deeprankcore.models.operations.parsers.vdwparam import VanderwaalsParam, ParamParser
+
+
+_log = logging.getLogger(__name__)
+
+_forcefield_directory_path = os.path.realpath(os.path.join(os.path.dirname(__file__), '../../../domain/forcefield'))
```

Comment:
is there a cleaner/prettier way to define this path?

Reply:
I looked a bit but being the file nested in domain/forcefield other ways are very similar to this. Since we removed one level for operations (deleting models) we have one less `..` now

Reply:
It's more that I don't like the fact that if we change something in the package structure we need to change this manually. Was wondering whether there is some nice pythonic way to read the folder directly, similar to an import. But probably not worthwhile investigating it if you don't have a ready to go solution.